client/client.go

```diff
@@ -1,18 +1,41 @@
 package main
 
 import (
-	"fmt"
+	"bufio"
+    "fmt"
+    "os"
 	"log"
+	"math/big"
+	"encoding/json"
 
 	"github.com/gorilla/websocket"
 )
```

Comment:
Something 'not yes' with formatting here. Did you correctly setup Golang plugin in your VSCode?

Reply:
fixed